features/account/customer_account/address_book/setting_default_address.feature

```diff
@@ -21,14 +21,14 @@ Feature: Marking an address as default
         Given I am browsing my address book
         When I set the address of "Lucifer Morningstar" as default
         Then I should be notified that the address has been set as default
-        And it should be marked as my default address
         And I should have 1 address in my address book
+        And address "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States", "Arkansas" should be marked as my default address
 
     @ui
     Scenario: Only one address can be default
-        Given I am browsing my address book
-        And my default address is of "Lucifer Morningstar"
+        Given my default address is of "Lucifer Morningstar"
```

Comment:
Also, why do we need? We have declared this address as default in the last step of background